### PR TITLE
[platform] Fix less than minute duration formatting

### LINKED ISSUE #9775: [core] Duration formatter returns empty string for duration 1...60 seconds
It should return the "0min" string.

### PR DESCRIPTION
**Pull request by kirylkaveryn**
Closes #9775

### CODE REVIEW

File: platform/duration.cpp [hunk 1]
```diff
@@ -84,7 +84,7 @@ std::string Duration::GetString(std::initializer_list<Units> units, std::string_
   ASSERT(units.size(), ());
   ASSERT(IsUnitsOrderValid(units), ());
 
-  if (m_seconds.count() == 0)
+  if (SecondsToUnits(m_seconds, Units::Minutes) == 0)
```
Comment (biodranik) [inline platform/duration.cpp:87]: Is there a test for this case?

**Review by biodranik — APPROVED** [review]

File: platform/duration.cpp (hunk 1 above)
Comment (kirylkaveryn) [reply platform/duration.cpp:87]: No there are no tests for this case. I'll add one

**Comment by vng** [comment]
Add some fire :) What string do we expect for 59 seconds? 0min or 1min ?

**Comment by kirylkaveryn** [comment]
> Add some fire :) What string do we expect for 59 seconds? 0min or 1min ?

Such conversion `duration_cast<minutes>(59 sec).count();` will make as 0 min... 
Maybe it will be enough for such a rare case as a track with a duration of less than 1 minute? 
I think that both 0min and 1min will be ok for such cases.

**Pull request merged**